source/_posts/2019-10-30-release-101.markdown

```diff
@@ -1705,7 +1705,7 @@ The **holiday name** sensor has been renamed to **holiday**. ([@tsvi] - [#27654]
 [lastfm docs]: /integrations/lastfm/
 [lg_soundbar docs]: /integrations/lg_soundbar/
 [lifx docs]: /integrations/lifx/
-[lifx_legacy docs]: /integrations/lifx_legacy/
+[lifx_legacy docs]: /integrations/lifx/
```

Comment:
We actually have a lifx_legacy integration, the docs seem to be missing.

Assigning myself to figure out what it is, does, and if it should stay or not.

Reply:
Yeah, I saw that too.
However, the `lifx_legacy` integration is referenced in the second line of the [lifx docs](https://www.home-assistant.io/integrations/lifx/). I thought that maybe the 2 integrations shared the same docs.